spec/features/animal-services/animal-services.automater.js

```diff
@@ -0,0 +1,98 @@
+const webdriver = require('selenium-webdriver');
+
+const By = webdriver.By;
+
+// //a[contains(text(), 'About Google')]
```

Comment:
Remove?

Reply:
removed, @mxsnyder I slightly changed updated the code, to handle rescued pets, please review again.